src/xpcc/architecture/platform/devices/avr/at9032_64_128-can.xml

```diff
@@ -26,7 +26,7 @@
       <parameter device-name="64|128" name="ram_length">3072</parameter>
       <parameter device-name="64|128" name="ram_block_length">32</parameter>
     </driver>
-    <driver type="adc" name="at90_tiny_mega"/>
+    <driver type="adc" name="at90"/>
```

Comment:
was this introduced in commit 49154d5e39043447d068019157bbec03cd65718b?

If so, could you please rebase and merge the `fixup` commits?